src/main/java/com/agilethought/atsceapi/validator/history/HistoryValidator.java

```diff
@@ -106,6 +107,7 @@ private void validateFixedExpenses(FixedExpenses fixedExpenses, List<ErrorDetail
 				+ (fixedExpenses.getInternet() == null ? 0 : fixedExpenses.getInternet())
 				+ (fixedExpenses.getFeed() == null ? 0 : fixedExpenses.getFeed())
 				+ (fixedExpenses.getOthers() == null ? 0 : fixedExpenses.getOthers());
+		System.out.println("total: " + fixedExpenses.getTotal() + " - calculated: " + calculatedTotal);

```

Comment:
It's a BAD practice use System.out. methods. please change it for log4j implementation.